skfda/exploratory/visualization/_outliergram.py

```diff
@@ -0,0 +1,152 @@
+        https://academic.oup.com/biostatistics/article/15/4/603/266279
+    """
+
+    def __init__(
+        self,
+        fdata: FDataGrid,
+    ) -> None:
+        self.fdata = fdata
+        self.depth = ModifiedBandDepth()
+        self.depth.fit(fdata)
+        self.mbd = self.depth(fdata)
+        self.mei = self.modified_epigraph_index_list()
+
+    def plot(
+        self,
+        chart: Optional[S] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[List[Axes]] = None,
+        n_rows: Optional[int] = None,
+        n_cols: Optional[int] = None,
+        **kwargs,
+    ) -> Figure:
+        """
+        Plot Outliergram.
+
+        Plots the Modified Band Depth (MBD) on the Y axis and the Modified
+        Epigraph Index (MEI) on the X axis. This points will create the form of
+        a parabola. The shape outliers will be the points that appear far from
+        this curve.
+        Args:
+            chart (figure object, axe or list of axes, optional): figure over
+                with the graphs are plotted or axis over where the graphs are
+                plotted. If None and ax is also None, the figure is
+                initialized.
+            fig (figure object, optional): figure over with the graphs are
+                plotted in case ax is not specified. If None and ax is also
+                None, the figure is initialized.
+            axes (list of axis objects, optional): axis where the graphs
+                are plotted. If None, see param fig.
+            n_rows (int, optional): designates the number of rows of the figure
+                to plot the different dimensions of the image. Only specified
+                if fig and ax are None.
+            n_cols(int, optional): designates the number of columns of the
+                figure to plot the different dimensions of the image. Only
+                specified if fig and ax are None.
+            kwargs: if dim_domain is 1, keyword arguments to be passed to the
+                matplotlib.pyplot.plot function; if dim_domain is 2, keyword
+                arguments to be passed to the matplotlib.pyplot.plot_surface
+                function.
+        Returns:
+            fig (figure object): figure object in which the depths will be
+            scattered.
+        """
+        fig, axes = _get_figure_and_axes(chart, fig, axes)
+        fig, axes = _set_figure_layout_for_fdata(
+            self.fdata, fig, axes, n_rows, n_cols,
+        )
+
+        axe = axes[0]
+
+        axe.scatter(
+            self.mei,
+            self.mbd,
+            **kwargs,
+        )
+
+        # Set labels of graph
+        fig.suptitle("Outliergram")
+        axe.set_xlabel("MEI")
+        axe.set_ylabel("MBD")
+        axe.set_xlim([0, 1])
+        axe.set_ylim([
+            self.depth.min,
+            self.depth.max,
+        ])
+
+        return fig
+
+    def modified_epigraph_index_list(self) -> List[float]:
```

Comment:
You are returning a numpy `ndarray`, right?

Reply:
yes, changed